### DISCUSSION TITLE
Git History Show

### DISCUSSION THREAD

**Discussion by bobo541574** [zed-industries/zed#9387] [Feature Requests]
Hello, Zed team. Please...
Could you add a feature like Git Len Extension (git commit history right side of code line in VS code)?

**Comment by bennetbo** [comment]
Sounds like you are looking for #4367 which is tracked in #8665

**Reply by Moshyfawn** [reply]
I believe #4793 is the direct feature request.

**Comment by JosephTLyons** [comment]
Zed now has both git blame in the gutter and inline blame.

- https://github.com/zed-industries/zed/issues/4793

**Reply by KaKi87** [reply]
What about showing the project git log though ?

**Reply by Mooling0602** [reply]
> Zed 现在既有边框中的 git blame，也有内联 blame。
> 
> * [Git in-line blame indicators #4793](https://github.com/zed-industries/zed/issues/4793)

I think this is actually we need, more like `git log`:
> Screenshot from VSCode. I'm using zh_cn.

<img width="522" height="683" alt="屏幕截图_20251220_201733" src="https://github.com/user-attachments/assets/3b91fda5-728c-4c4d-978a-8cfc7f1e21f2" />

**Reply by KaKi87** [reply]
Please copy JetBrains though, not VSCode. The latter's Git integration lacks so much.

**Reply by Mooling0602** [reply]
> Please copy JetBrains though, not VSCode. The latter's Git integration lacks so much.

Actually I agree with you.

**Comment by iTrooz** [comment]
Maybe an issue should be created for this specific feature request ? Since none linked so far seem to be about this exactly

**Reply by InvictusNavarchus** [reply]
> Maybe an issue should be created for this specific feature request ? Since none linked so far seem to be about this exactly

@iTrooz I think this one is #26511. This one  #44434 implements it. Good thing is, it's been merged. So we'll just need to wait for release. Although from the screenshot, it doesn't look sidebar. But we can create a new discussion after release.
